data/decks/mole.json

```diff
@@ -0,0 +1,238 @@
+{
+	"id": "mole",
+	"season": 4,
+	"display_name": "Mole Knight",
+	"cardback": "cardback_mole.jpg",
+	"exceed_cost": 4,
+	"buddy_card": "burrow",
+	"buddy_display_name": "Burrow",
+	"buddy_exceeds": true,
+	"buddy_starting_offset": 0,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "place_buddy_at_range",
+			"buddy_name": "Burrow",
+			"range_min": 3,
+			"range_max": 3,
+			"and": {
+				"effect_type": "draw",
+				"amount": 1
+			}
+		}
+	}],
+	"character_action_exceeded": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "place_buddy_at_range",
+			"buddy_name": "Burrow",
+			"range_min": 2,
+			"range_max": 4,
+			"and": {
+				"effect_type": "draw",
+				"amount": 1
+			}
+		}
+	}],
+	"ability_effects": [
+		{
+			"timing": "set_strike",
+			"condition": "initiated_strike",
+			"effect_type": "choice",
+			"choice": [
+				{
+					"effect_type": "set_used_character_bonus",
```

Comment:
Wouldn't add_attack_effect work? Like Kokonoe?

Reply:
Should be the same thing; i guess it avoids the need to repeat the effect